ci-operator/step-registry/openstack/conf/resourcenames/openstack-conf-resourcenames-commands.sh

```diff
@@ -35,7 +35,7 @@ declare -A compute_flavor=(
 	)
 
 declare -A compute_azs=(
-	['openstack-kuryr']=''
+	['openstack-kuryr']='az0'
```

Comment:
I'm confused as to why `openstack-kuryr` is a separate cluster profile. What prevents us to point to an existing one, like `openstack-vh-mecha-az0` and set the `NETWORK_TYPE` to `Kuryr`?
@MaysaMacedo perhaps you know the history of the `openstack-kuryr` cluster profile.

Reply:
yeah i think we could move the kuryr jobs to this profile. Historically we did that so we could set a quota on Kuryr jobs, since they were sharing mecha-central resources.

Reply:
yes, it was related to quotas e avoiding possible impacts on other clusters running at the same moment for other test scenarios